manifests/m/Microsoft/OpenJDK/11/11.0.10.9/Microsoft.OpenJDK.11.yaml

```diff
@@ -0,0 +1,29 @@
+﻿# yaml-language-server: $schema=https://aka.ms/winget-manifest.singleton.1.0.0.schema.json
+PackageIdentifier: Microsoft.OpenJDK.11
+Publisher: Microsoft
+PackageName: Microsoft OpenJDK 11 Preview
+Author: Microsoft
+ShortDescription: The Microsoft Build of OpenJDK is a new no-cost long-term supported distribution and Microsoft"s new way to collaborate and contribute to the Java ecosystem.
+Moniker: ms-openjdk-11
+Tags:
+- Java
+- OpenJDK
+- "11"
+- LTS
+PackageUrl: https://www.microsoft.com/openjdk
+License: GPL2 with Classpath Exception.
+LicenseUrl: https://openjdk.java.net/legal/gplv2+ce.html
+MinimumOSVersion: 10.0.0.0
```

Comment:
Will `winget` work with other versions of Windows? Or only 10?

Reply:
Currently, only Windows 10.

Reply:
Ok got it.